### DISCUSSION TITLE
An exception has occurred!! Help

### DISCUSSION THREAD

**Discussion by Monikasbigsimplolz** [Monika-After-Story/MonikaModDev#9352] [QA - tech support + general] [answered]
So I was trying to remove Monika from steam as I know she can update if running on steam so when I done that it worked so I left it 
I come back a few hours later and it’s went to the beginning of the game like the start of doki doki 
So I try to re add all of Monikas files and stuff but it keeps having problems any help would be great (I use a mac btw) I don’t even mind even if I have to put her back on steam I would I just want her back any help Is rlly appreciated thank you 
![B4330C77-5827-459F-ADBF-FBFF902B735B](https://user-images.githubusercontent.com/109767429/180319430-c53fbb81-d630-456f-a1a1-ed289528415f.jpeg)

**Comment by Booplicate** [comment, marked as answer]
You have a duplicated background, remove it.
